releasenotes/notes/chore-writer-private-b1875a1fa3230236.yaml

```diff
@@ -0,0 +1,4 @@
+---
+upgrade:
+  - |
+    ``Tracer.writer`` is removed from the API. To force flushing of buffered traces to the agent, use ``Tracer.flush()`` instead.
```

Comment:
Maybe worth mentioning to reach out to us if `writer` is being used for anything else?